Website/app/Features.js

```diff
@@ -0,0 +1,78 @@
+      <h1 className="text-center mb-6 font-extrabold text-4xl pt-6">
+        Features
+      </h1>
+      <div className="featuresContainer column-1 flex flex-wrap justify-center">
+        <div className="feature m-2 p-4 rounded-md h-max flex flex-col justify-center shadow-lg bg-slate-100 text-black w-60">
+          <i
+            className="fa fa-unlock-alt self-center mb-5 hover:text-lime-700"
+            aria-hidden="true"
+            style={{ fontSize: "3rem" }}
+          ></i>
+          <h3 className="feature--title text-lg mb-1">
+            It&apos;s Free and Open-Source
+          </h3>
+          <p className="feature--description font-normal leading-normal">
+            Drifty is both free and open-source, so, you can bring new features
+            that you wish to have 🎉
+          </p>
+        </div>
+
+        <div className="feature m-2 p-4 rounded-md h-max flex flex-col justify-center shadow-lg bg-slate-100	 text-black w-60">
+          <i
+            className="fa fa-download self-center mb-5 hover:text-sky-400"
+            aria-hidden="true"
+            style={{ fontSize: "3rem" }}
+          ></i>
+
+          <h3 className="feature--title text-lg  mb-1">
+            Faster Downloading of files
+          </h3>
+          <p className="feature--description font-normal leading-normal">
+            Drifty supports parallel downloading using multiple threads
+          </p>
+        </div>
+        <div className="feature m-2 p-4 rounded-md h-max flex flex-col justify-center shadow-lg bg-slate-100 text-black w-60">
+          <i
+            className="fa fa-youtube-play self-center mb-5 hover:text-red-600"
+            aria-hidden="true"
+            style={{ fontSize: "3rem" }}
+          ></i>
+
+          <h3 className="feature--title text-lg  mb-1">
+            Support for downloading videos from YouTube, Instagram, etc.
+          </h3>
+          <p className="feature--description font-normal leading-normal">
+            Drifty allows downloading videos from popular platforms like YouTube
+            and Instagram with ease.
+          </p>
+        </div>
+        <div className="feature m-2 p-4 rounded-md h-max flex flex-col justify-center shadow-lg bg-slate-100	 text-black w-60">
+          <i
+            className="fa fa-desktop self-center mb-5 hover:text-gray-400"
+            aria-hidden="true"
+            style={{ fontSize: "3rem" }}
+          ></i>
+          <h3 className="feature--title text-lg mb-1">
+            Available both in GUI and CLI mode
+          </h3>
+          <p className="feature--description font-normal leading-normal">
+            Drifty has both CLI and GUI mode. So, it can be used anywhere as CLI
+            or as GUI according to the requirements of the user.
+          </p>
+        </div>
+      </div>
+      <div className="text-2xl font-bold flex justify-center my-4">
+        ....and many more!
+      </div>
+    </div>
+  );
+}
+
+export default Features;
```

Comment:
@anevski-stefan Some changes are requested:
- I think we can remove the **Font Awesome** module because it is already imported as a script in `layout.js` file.
- The cards in the features section can be simplified by using `arrays` and `map()` function.
- Add hover effects to the whole card instead of only to the icons.
---
You can commit the below change to accept the changes I mentioned above.
```suggestion
import React from "react";

function Features() {
  const features = [
    {
        title: "It's Free and Open-Source",
        description: "Drifty is both free and open-source, so, you can bring new features that you wish to have 🎉",
        icon: "fa-brands fa-osi",
        color: "text-lime-600",
    },
    {
        title: "Faster Downloading of files",
        description: "Drifty supports parallel downloading using multiple threads",
        icon: "fa fa-download",
        color: "text-sky-400",
    },
    {
        title: "Support for downloading videos from YouTube, Instagram, etc.",
        description: "Drifty allows downloading videos from popular platforms like YouTube and Instagram with ease.",
        icon: "fa fa-youtube-play",
        color: "text-red-600",
    },
    {
        title: "Available both in GUI and CLI mode",
        description: "Drifty has both CLI and GUI mode. So, it can be used anywhere as CLI or as GUI according to the requirements of the user.",
        icon: "fa fa-desktop",
        color: "text-gray-700",
    },
  ];
  return (
    <div className="py-6 px-12 background bg-gradient-to-r from-cyan-500 to-blue-500 text-white font-sans select-none">
      <h1 className="text-center mb-6 font-extrabold text-4xl pt-6">
        Features
      </h1>
      <div className="column-1 flex flex-wrap justify-center gap-1">
        {features.map((feature, index) => (
            <div key={index} className="m-2 p-4 rounded-md h-max flex flex-col justify-center shadow-lg bg-slate-100 font-bold text-black w-60 hover:-translate-y-1 hover:scale-110 duration-300">
                <i
                    className={feature.icon + " self-center mb-5 " + feature.color}
                    aria-hidden="true"
                    style={{ fontSize: "3rem" }}
                ></i>
                <h3 className="text-lg mb-1">
                    {feature.title}
                </h3>
                <p className="font-normal leading-normal">
                    {feature.description}
                </p>
            </div>
        ))}
      </div>
      <div className="text-2xl font-bold flex justify-center my-4">
        ....and many more!
      </div>
    </div>
  );
}

export default Features;
```